### DISCUSSION TITLE
util.inherits is not a function

### DISCUSSION THREAD

**Discussion by mxvsh** [oven-sh/bun#7435] [Q&A]
I am getting `util.inherits is not a function` after I try to run build files using `bun run`. I have monorepo, when I remove the use of the package from the monorepo, the error goes away.

```sh
❯ bun run build/index.js
8152 |           _this.close();
8153 |         }
8154 |       }
8155 |     });
8156 |   };
8157 |   util.inherits(TransportStream, Writable);
      ^
TypeError: util.inherits is not a function. (In 'util.inherits(TransportStream, Writable)', 'util.inherits' is undefined)
```

**Comment by yonixw** [comment]
Same for me, monorepo with workspaces in the main package.json

It worked for me after adding `--target=bun` (in windows, bun v1.2.1). I guess the reason is bun silently defaults browser target.

![image](https://github.com/user-attachments/assets/1b78926c-37bb-4222-849d-1e27bcf5da7f)

**Comment by graniteviper** [comment]
Hello, I am facing the same issue. While building and starting an express server it says util.inherits is not a function. (In 'util.inherits(SendStream, Stream)', 'util.inherits' is undefined). It will be great if you can tell, how to solve the issue. I tried adding --target=bun in the start command but it didn't work for me.
